packages/math-input/src/components/echo-manager.tsx

```diff
@@ -56,7 +52,6 @@ class Echo extends React.Component {
         return (
             <div style={containerStyle}>
                 <KeypadButton
-                    name={id}
```

Comment:
Logic change: Didn't see that KeypadButton ever used this and TS was complaining.